### ISSUE TITLE
Inconsistent Tensor Size

### ISSUE THREAD

**Issue by yu-changqian** [wkentaro/pytorch-fcn#5]
When I implement the VOC2012 Dataset based your code, it occurs an error "incosistent tensor size". The code about dataset is same with yours. But I add some test code as below:
```
if __name__ == '__main__':
    dst = VOCDataSet("./data", is_transform=True)
    trainloader = data.DataLoader(dst, batch_size=4)
    for i, data in enumerate(trainloader):
        imgs, labels = data
        print(imgs[0].type())
```
Did you encounter this problem?

**Comment by wkentaro** [comment]
Currently there's not image resizing or cropping for batch training in my code, so you need to set batch_size=1.
If you'd like to try batch training, you need to crop/resize/centerize images to have same size because VOCDataset contains various sized images.

**Comment by yu-changqian** [comment]
Ok. It works.Thank you.So I have to implement a co-transform for the input and target images and then I can set the batch_size = 4(any just not 1 )?

**Comment by wkentaro** [comment]
Yeah. Note that you need to use `NEAREST` for resizing label image, not `BILINEAR`.

**Comment by yu-changqian** [comment]
Why? How different about these ways?

**Comment by wkentaro** [comment]
Label image has categorical values, not continuous, so we need to interpolate with nearest values at transforming.

**Comment by yu-changqian** [comment]
OK. I get it. Thank you very much for your help.
